docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,232 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Requirements
+
+You must log on to the VEBA appliance as root. You can do this from the console. If you want SSH access, execute the following command:
+
+```
+systemctl start sshd 
+```
+
+This turns on the SSH daemon but does not enable it to start on appliance boot. You should now be able to SSH into the VEBA appliance. 
+
+<BR>
+
+## Troubleshooting an initial deployment
+
+If VEBA is not working immediately after deployment, the first thing to do is check your Kubernetes pods. 
+
+```
+kubectl get pods -A
+```
+
+Here is the command output:
+
+```
+NAMESPACE        NAME                                   READY   STATUS          RESTARTS        AGE
+kube-system      coredns-584795fc57-hcvxh               1/1     Running              1          4d15h
+kube-system      coredns-584795fc57-hf72w               1/1     Running              1          4d15h
+kube-system      etcd-veba02                            1/1     Running              1          4d15h
+kube-system      kube-apiserver-veba02                  1/1     Running              1          4d15h
+kube-system      kube-controller-manager-veba02         1/1     Running              1          4d15h
+kube-system      kube-proxy-fj47p                       1/1     Running              1          4d15h
+kube-system      kube-scheduler-veba02                  1/1     Running              1          4d15h
+kube-system      weave-net-vs8ls                        2/2     Running              4          4d15h
+projectcontour   contour-5cddfc8f6-8hzd6                1/1     Running              1          4d15h
+projectcontour   contour-5cddfc8f6-jq7d8                1/1     Running              1          4d15h
+projectcontour   contour-certgen-f92l5                  0/1     Completed            0          4d15h
+projectcontour   envoy-gcmqt                            1/1     Running              1          4d15h
+vmware           tinywww-7fcfc6fb94-mfltm               1/1     Running              1          4d15h
+vmware           vmware-event-router-5dd9c8f858-5c9mh   0/1     CrashLoopBackoff     6          4d13h
+```
+
+One of the first things to look for is whether a pod is in a crash state. In this case, the vmware-event-router pod is crashing. We need to look at the logs with this command:
+
+```
+kubectl logs vmware-event-router-5dd9c8f858-5c9mh  -n vmware
```

Comment:
Add a note that the name of the pod suffix `-5dd9c8f858-5c9mh` will be different in each environment.

Reply:
Added